### ISSUE TITLE
Entity generator doesn't generate values for all supported options

### ISSUE THREAD

**Issue by lcp0578** [doctrine/doctrine2#6703]
in symfony 3.2.12 , doctrine 2.5, doctrine bundle 1.6
```
php bin/console doctrine:mapping:import --force KitAdminBundle yml  --filter="UserTest"
Importing mapping information from "default" entity manager
  > writing D:\xampp\htdocs\kitlabs\KitPHP\src\KitAdminBundle/Resources/config/doctrine/UserTest.orm.yml
```
UserTest.orm.yml
```
KitAdminBundle\Entity\UserTest:
    type: entity
    table: user_test
    id:
        id:
            type: integer
            nullable: false
            options:
                unsigned: false
            id: true
            generator:
                strategy: IDENTITY
    fields:
        username:
            type: string
            nullable: false
            length: 32
            options:
                fixed: false
        mobile:
            type: string
            nullable: false
            length: 11
            options:
                fixed: true  # this is true
```
convert annotation
```
php bin/console doctrine:mapping:convert annotation ./src --filter=UserTest
Processing entity "KitAdminBundle\Entity\UserTest"
```
UserTest.php
```
class UserTest
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="username", type="string", length=32, nullable=false)
     */
    private $username;

    /**
     * @var string
     *
     * @ORM\Column(name="mobile", type="string", length=11, nullable=false)
     */
    private $mobile;
}
```
but @ORM\Column(name="mobile", type="string", length=11, nullable=false) not **filxed=true**
thx

**Comment by lcobucci** [comment]
@lcp0578 handled in #6849. It will be available on `v2.6.x` only.

**Comment by lcp0578** [comment]
@lcobucci thx
